sentry-actix/src/lib.rs

```diff
@@ -239,26 +238,20 @@ fn should_capture_request_body(
 }
 
 /// Extract a body from the HTTP request
-async fn body_from_http(req: &mut ServiceRequest) -> Result<BytesMut, PayloadError> {
-    let mut stream = req.take_payload();
+async fn body_from_http(req: &mut ServiceRequest) -> actix_web::Result<Bytes> {
+    let stream = req.extract::<actix_web::web::Payload>().await?;
+    let body = stream.try_collect::<BytesMut>().await?.freeze();
```

Comment:
it's possible to do this but it's not semantically equivalent due to [`PayloadConfig::limit` being applied](https://docs.rs/actix-web/4.10.2/actix_web/web/struct.Bytes.html#impl-FromRequest-for-Bytes)

```suggestion
    let stream = req.extract::<Bytes>().await?;
```